selenium/selenium/ex10.rb

```diff
@@ -0,0 +1,29 @@
+require 'selenium-webdriver'
+
+
+Selenium::WebDriver::Chrome::Service.driver_path='G:/selenium training/drivers/chromedriver.exe'
+driver=Selenium::WebDriver.for :chrome
+ 
+
+
+col = 1
+
+driver.get "https://computer-database.gatling.io/computers"
+
+table = driver.find_element(:tag_name, "table")
+
+table_head = table.find_element(:tag_name, "thead")
+
+header_row = table_head.find_element(:tag_name, "tr")
+
+# puts header_row.text
+
+header_row_data = header_row.find_elements(:tag_name, "th")
+
+puts header_row_data[col].text
```

Comment:
You do not need to fetch the header row separately. You only need to read the `tr` elements of the table which will also fetch the header values. as they too are enclosed within a `tr` tag.